### DISCUSSION TITLE
Primary key column with unique identifier.

### DISCUSSION THREAD

**Discussion by baskarkanini** [Azure/data-api-builder#1750] [Q&A] [answered]
I have a table name Building with below schema.
CREATE TABLE [dbo].[Building](
	[BuildingId] [uniqueidentifier] NOT NULL,
	[LocationId] [uniqueidentifier] NOT NULL,
	[BuildingName] [varchar](255) NOT NULL)
	
During the post operation in swagger i dont want to send the Building ID. Since it is a primary key with default NewID() generation in it.
How to ignore this column in the post operation. how to update my DAB configuration with this operation.
{
  "BuildingId": (newid()),  //completely remove it.. 
  "LocationId": "85115AFA-7847-4E32-B01D-13181EC656D5",
  "BuildingName": "TEST@@@@@"
}

**Comment by seantleonard** [comment]
Hi @baskarkanini , 

The ability to ignore the primary key field in the POST request body is a new feature in our latest **pre-release version (0.9.5-rc)** that you can find https://github.com/Azure/data-api-builder/releases/tag/v0.9.5-rc

Please take a look and see if that meets your needs. If not, please let us know.

**Reply by baskarkanini** [reply]
Thanks Seantleonard

**Reply by Aniruddh25** [reply, marked as answer]
Hi @baskarkanini,

Specifically, after installing this new release candidate, could you try adding the following to your configuration under the `runtime.rest` section?

```
"runtime": { 
      "rest": {
          "request-body-strict":  false
       }
}
```

**Reply by baskarkanini** [reply]
Thanks Aniruddh
